commons/constants.py

```diff
@@ -209,9 +209,7 @@ class Rest:
     S3_ACCOUNTS = "s3_accounts"
     ACC_NAME = "account_name"
     ACC_EMAIL = "account_email"
-    SECRET_KEY = "secret_key"
```

Comment:
did we check if these are not getting used by anyone else other than CSM files?

Reply:
yes, they are not getting except the place it is changed